.ci/run_e2e_tests.sh

```diff
@@ -1,5 +1,14 @@
 SCRIPTS_PATH=$(cd $(dirname "${BASH_SOURCE[0]}"); pwd)
 SRC_PATH=$(dirname "${SCRIPTS_PATH}")
+
+# Early auth to avoid limit rating
+# If Docker Credentials are not available skip the Byzantine Tests
```

Comment:
Could you remove the second comment here as I think it's intended for the place below where we actually set byzantine image if the docker credentials are not as opposed to the authentication we're doing here?

Reply:
Done :)